docs/tutorials/02_ui/viz_accordion_text_ui.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+"""
+================================================
+Example to demonstrate the use of Accordion2D UI
```

Comment:
What is the difference from the previous one? You need to add more information to the description.